src/server.ts

```diff
@@ -423,7 +424,7 @@ export class SubscriptionServer {
               query: parsedMessage.payload.query,
               variables: parsedMessage.payload.variables,
               operationName: parsedMessage.payload.operationName,
-              context: Object.assign({}, isObject(initResult) ? initResult : {}),
+              context: Object.assign({}, isObject(initResult) ? initResult : {}, parsedMessage.payload.context),
```

Comment:
Why is context sent over the network?
@urigo @helfer @stubilo - Thats a serious security issue, you shall not release the package with that